# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchStreamUtils.h
# Commit: 4fd92351d64b
# Full Hash: 4fd92351d64b1cbeb82beaf6228b21c01dc9bc03
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 7a4e3f5f674a -r 4fd92351d64b dom/fetch/FetchStreamUtils.h
--- a/dom/fetch/FetchStreamUtils.h	Wed Jan 18 00:50:19 2023 +0000
+++ b/dom/fetch/FetchStreamUtils.h	Wed Jan 18 00:50:20 2023 +0000
@@ -40,7 +40,8 @@
 // process. Can only be called in the parent process.
 ParentToChildStream ToParentToChildStream(
     const NotNull<nsCOMPtr<nsIInputStream>>& aStream, int64_t aStreamSize,
-    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent);
+    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent,
+    bool aSerializeAsLazy = true);
 
 // Convert a ParentToParentStream to a ParentToChildStream. Can only be called
 // in the parent process.